spring-cloud-config-monitor/src/main/java/org/springframework/cloud/config/monitor/BitbucketPropertyPathNotificationExtractor.java

```diff
@@ -59,6 +58,14 @@ else if (("repo:refs_changed".equals(headers.getFirst("X-Event-Key"))
 				return new PropertyPathNotification("application.yml");
 			}
 		}
+		else if ("pr:merged".equals(headers.getFirst("X-Event-Key"))
```

Comment:
Rather than adding a whole new section here, why not check for the presence of the "changes" key in the existing section?

Reply:
I am removing the check for "changes" key for "pr:merged" only

Reply:
:+1: